Sources/App/Classes/Dialogs/TDCServerPropertiesSheet.m

```diff
@@ -2305,6 +2305,15 @@ - (BOOL)tableView:(NSTableView *)tableView acceptDrop:(id <NSDraggingInfo>)info
 	return YES;
 }
 
+- (void)tableView:(NSTableView *)tableView mouseDownInHeaderOfTableColumn:(NSTableColumn *)tableColumn {
+	NSSortDescriptor *sortDescriptor = [[NSSortDescriptor alloc] initWithKey:nil ascending:YES comparator:^NSComparisonResult(IRCChannelConfig *cfg1, IRCChannelConfig *cfg2) {
```

Comment:
i dont know if i have to `dealloc` at the end of the function...

Reply:
Textual uses [Automatic Reference Counting](https://en.wikipedia.org/wiki/Automatic_Reference_Counting) which is fancy speak for it not requiring `dealloc`. The compiler adds that for you.